config/forms/2021_2022.json

```diff
@@ -654,15 +654,18 @@
                   "hint_text": "",
                   "type": "radio",
                   "answer_options": {
-                    "0": {
-                      "value": "Renewal of fixed-term tenancy"
-                    },
                     "1": {
                       "value": "Internal transfer",
                       "hint": "Excluding renewals of a fixed-term tenancy"
                     },
+                    "10": {
+                      "value": "Previous tenant died with no succession"
+                    },
                     "2": {
-                      "value": "Relet to tenant who occupied same property as temporary accommodation"
+                      "value": "Re-let to tenant who occupied same property as temporary accommodation"
```

Comment:
This will need updating here too: https://github.com/communitiesuk/mhclg-data-collection-beta/blob/main/app/models/constants/case_log.rb#L176